src/auth/wire/token.clj

```diff
@@ -0,0 +1,14 @@
+(ns auth.wire.token
+  (:require [schema.core :as s]
+            [common-labsoft.schema :as schema]))
+
+(def token-types #{:customer :merchant :carrier :service})
```

Comment:
Should't this token type be #{:user :service}

And :customer :merchant :carrier be in the scopes?

Reply:
We have an auth interceptors that verifies the token type, we can use this to filter by category, but you comment is valid, just a different approach. If you think we can gather more value from using just user and service we can go with it